conda_build/environ.py

```diff
@@ -118,7 +145,9 @@ def get_dict(m=None, prefix=None):
             d['CPU_COUNT'] = "1"
 
     if m.get_value('source/git_url'):
-        d.update(**get_git_build_info(d['SRC_DIR']))
+        d.update(**get_git_build_info(d['SRC_DIR'],
+                                      m.get_value('source/git_url'),
+                                      m.get_value('source/git_rev', default='master')))
```

Comment:
Thanks for pointing that out.  I just want to make sure: this fix works for people who specify git_tag in their recipe, rather than git_rev?